src/AppBundle/Resources/config/content_view.yml

```diff
@@ -3,17 +3,28 @@ frontend_group:
         full:
             ng_category:
                 template: "@ezdesign/content/full/ng_category.html.twig"
-                controller: "ngsite.controller.full_view:viewNgCategory"
+                controller: "ngsite.controller.check_redirect"
                 match:
                     Identifier\ContentType: ng_category
+                queries:
+                    subtree:
+                        query_type: 'SiteAPI:Location/Subtree'
+                        max_per_page: '@=content.getFieldValue("page_limit").value > 0 ? content.getFieldValue("page_limit").value : viewParam("childrenLimit", 12)'
+                        page: '@=request.query.get("page", 1)'
```

Comment:
This can be replaced with `@=queryParam("page", 1)`

Reply:
@pspanja Sorry, I missed that one in the docs. Fixed in https://github.com/netgen/media-site/pull/51/commits/b8b54927e380151577f63f3dbc37446803634868